src/software/world/game_state.h

```diff
@@ -357,4 +345,23 @@ class GameState
      *         in a state of play where the ball needs to be placed
      */
     std::optional<Point> getBallPlacementPoint() const;
+
+    State getState() const;
```

Comment:
Since these are only being used in game_state_test.cpp can we delete them? For equality tests, we should be able to accomplish the same thing with updateRefboxGameState

Reply:
deleted